easysid.cpp

```diff
@@ -16,6 +17,16 @@
 SID sid;
 PcmOutput *pcm = NULL;
 PcmOutput *wav = NULL;
+int g_sample_rate = 44100;
+
+bool initAudioOutput()
```

Comment:
I'm uncomfortable with the idea of this function taking no arguments and returning only a bool. I question the need for this function at all. It hides its functionality (changing a global variable) behind a facade of a parameterless function. Suggest removing it.

If you want to avoid repeating the code that deallocates pcm, consider using unique_ptr.